### PR TITLE
Create Posts “2021-08-03-update_to_our_customer_responsibility_matrix_crm”

### PR DESCRIPTION
**Pull request by pburkholder**
Automatically generated by Netlify CMS

## Security Considerations
N/A

### CODE REVIEW

**Comment by pburkholder** [comment]
https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/cms/posts/2021-08-03-update_to_our_customer_responsibility_matrix_crm/2021/08/03/update_to_our_customer_responsibility_matrix_crm/ is preview

File: _posts/2021-08-03-update_to_our_customer_responsibility_matrix_crm.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+date: August 3, 2021
+excerpt: Today's update to our Control Implementation Summary (CIS) + Customer
+  Responsibility Matrix (CRM) clarifies some language and inhertiance status for
+  controls.
+---
+The **[Control Implementation Summary (CIS) + Customer Responsibility Matrix (CRM) + Control-by-Control Inheritance (.xlsx)](https://cloud.gov/resources/cloud.gov-CIS-Worksheet.xlsx)** is a summary of each Low and Moderate security control and whether it is handled by cloud.gov, shared responsibility, or customer responsibility. It includes guidance on which controls a customer system can fully or partially inherit from cloud.gov.
+
+You can read more on how to start the ATO process with cloud.gov at [](https://cloud.gov/docs/overview/fedramp-tracker/)[our FedRAMP Authorized page](https://cloud.gov/docs/overview/fedramp-tracker/)
+
+Changes to CRM:
```
Comment (mheadd) [inline _posts/2021-08-03-update_to_our_customer_responsibility_matrix_crm.md]:
```suggestion
Recent changes to CRM:
```

**Comment by pburkholder** [comment]
@mheadd If I'm out can you please merge this?

**Review by mheadd — APPROVED** [review]
LGTM!

**Pull request merged**